hieradata/common.yaml

```diff
@@ -213,6 +213,7 @@ profile::jenkinscontroller::jcasc:
       osDiskSize: 90
       agentJavaBin: '/opt/jdk-11/bin/java'
       agentJavaOpts: '-XX:+PrintCommandLineFlags'
+      path: '~/.asdf/shims:~/.asdf/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin'
```

Comment:
/usr/**games**? ^^

Reply:
Yes :D It is the default Ubuntu 20.04 PATH (extracted in https://ci.jenkins.io/job/Infra/job/acceptance-tests/job/check-agent-availability/1736/console and confirmed on a local VM)